package.json

```diff
@@ -82,16 +82,18 @@
   ],
   "jspm": {
     "main": "js/bootstrap",
+    "ignore": ["dist/js/npm"],
+    "directories": {
+      "lib": "dist"
+    },
     "shim": {
       "js/bootstrap": {
-        "deps": "jquery",
+        "deps": ["jquery"],
         "exports": "$"
       }
     },
-    "files": [
-      "css",
-      "fonts",
-      "js"
-    ]
+    "dependencies": {
+      "jquery": "github:components/jquery"
```

Comment:
To use jspm's default jquery, whatever that is, try setting `"jquery": "*"` (or the jQuery version range - `"jquery": "^2.1.4"`). This way if we change the main jquery source package to something more official in due course, Bootstrap would naturally track that.